src/types/commitData.ts

```diff
@@ -6,3 +6,16 @@ export type GitHubCommitData = {
   }
   message: string
 }
+
+export type GitLocalDiskCommitData = {
+  sha: string
+} & GitHubCommitData
+
+// returned by Simple Git
+export type GitLocalDiskRawCommitData = {
```

Comment:
Just checking, are we able to use [simple-git's typings here](https://github.com/steveukx/git-js/blob/d184c13273abca4b6572c260f9625c19f944d4f7/simple-git/src/lib/tasks/log.ts#L39)?

Reply:
good point, will update to use `DefaultLogFields` type from simple-git

Reply:
done!